src/views/transactions/components/transaction-overview/transaction-overview.view.tsx

```diff
@@ -0,0 +1,337 @@
+// ToDo: Remove the disable of TS and the linter below once the component are migrated to TS
+/* eslint-disable */
+// @ts-nocheck
+import React from "react";
+import { useTheme } from "react-jss";
+import { BigNumber } from "ethers";
+
+import useTransactionOverviewStyles from "src/views/transactions/components/transaction-overview/transaction-overview.styles";
+import { getTokenAmountInPreferredCurrency, getFixedTokenAmount } from "src/utils/currencies";
+import TransactionInfo from "src/views/shared/transaction-info/transaction-info.view";
+import Container from "src/views/shared/container/container.view";
+import FiatAmount from "src/views/shared/fiat-amount/fiat-amount.view";
+import TokenBalance from "src/views/shared/token-balance/token-balance.view";
+import Spinner from "src/views/shared/spinner/spinner.view";
+import PrimaryButton from "src/views/shared/primary-button/primary-button.view";
+import { getRealFee } from "src/utils/fees";
+import Alert from "src/views/shared/alert/alert.view";
+import WithdrawInfoSidenav from "src/views/transactions/components/withdraw-info-sidenav/withdraw-info-sidenav.view";
+import { AsyncTask } from "src/utils/types";
+import { Theme } from "src/styles/theme";
+import { TxType } from "@hermeznetwork/hermezjs/src/enums";
+
+// domain
+import { HermezWallet, Account, FiatExchangeRates, Exit } from "src/domain/hermez";
+
+interface EstimatedWithdrawFee {
+  amount: string;
+  USD: number;
+}
+
+type Transaction =
+  | {
+      type: typeof TxType.Deposit;
+      amount: BigNumber;
+      account: Account;
+      onDeposit: (amount: BigNumber, account: Account) => void;
+    }
+  | {
+      type: typeof TxType.Transfer;
+      amount: BigNumber;
+      account: Account;
+      to: Partial<Account>;
```

Comment:
Can we avoid to have this `Partial<Account>` and have another `Account` here to be more domain driven?

Reply:
mmm... I'm not sure I follow. What do you mean by having another `Account`? Why would we need to declare another interface for an account? I'm probably missing something. Can you elaborate a bit more?